README.md

```diff
@@ -19,7 +19,12 @@ Here is an example of a Grafana dashboard for an ESP project. This dashboard rel
 
 ## Install a Released Version of the Plug-in
 
-Use the provided installation script to install the plug-in:
+An installation script is provided to install the plugin and configure Grafana. The installation script will modify the Grafana deployment by adding an environment variable to enable the plugin
```

Comment:
"plugin" should be "plug-in". We should use present tense if possible. Some sentences are a bit lengthy/complex. Below is my suggested revision, though I realize that you might need to make other changes to address the comments already added by Chris.

An installation script is provided to install the plug-in and configure Grafana. The installation script modifies the Grafana deployment by adding an environment variable. The environment variable enables the plug-in and configures a new `grafana.ini` file to enable UAA authentication.

The installation script configures Grafana as an OAuth client with the supported OAuth provider (UAA). Users of Grafana are directed to use the OAuth login page.